### ISSUE TITLE
False positive MissingAccountIdentifier if ACCT_USER_ID=-1 or ACCT_GROUP_ID=-1

### ISSUE THREAD

**Issue by NexAdn** [pkgcore/pkgcheck#327]
System: Recent Gentoo Linux with dev-util/pkgcheck-0.10.0

When run inside an overlay with `acct-group` and `acct-user` packages (where user/group IDs are always `-1`), pkgcheck complains with the following warning:

```
acct-group/pkg
  MissingAccountIdentifier: version 1: unable to determine the value of ACCT_GROUP_ID variable

acct-user/pkg
  MissingAccountIdentifier: version 1: unable to determine the value of ACCT_USER_ID variable
```

with simple ebuilds like this (which are–apart from the description, ID and ACCT_USER_GROUPS–exactly like the template vim provides when initially creating a new ebuild of that kind):

```bash
# Copyright 2021 Gentoo Authors
# Distributed under the terms of the GNU General Public License v2

EAPI=7

inherit acct-user

DESCRIPTION="user for cat/pkg"
ACCT_USER_ID=-1
ACCT_USER_GROUPS=( pkg )

acct-user_add_deps
```
```bash
# Copyright 2021 Gentoo Authors
# Distributed under the terms of the GNU General Public License v2

EAPI=7

inherit acct-group

ACCT_GROUP_ID=-1
```

I don't know if a warning in such a case is wanted or not, but I think “unable to determine the value of ... variable” is the wrong message to notify the user that the ID is negative.
